debezium-embedded/src/test/java/io/debezium/embedded/AbstractConnectorTest.java

```diff
@@ -101,10 +102,11 @@ public abstract class AbstractConnectorTest implements Testing {
     private static final String TEST_PROPERTY_PREFIX = "debezium.test.";
 
     private ExecutorService executor;
-    protected EmbeddedEngine engine;
+    protected TestingDebeziumEngine engine;
     protected BlockingQueue<SourceRecord> consumedLines;
     protected long pollTimeoutInMs = TimeUnit.SECONDS.toMillis(10);
     protected final Logger logger = LoggerFactory.getLogger(getClass());
+    protected final AtomicBoolean isRunning = new AtomicBoolean(false);
```

Comment:
Please rename to `isEngineRunning` or to `isConnectorRunning` to make sure it is not confused with the notion of running test.

Reply:
done